hspec-hedgehog.cabal

```diff
@@ -1,6 +1,6 @@
 cabal-version: 1.12
 name:           hspec-hedgehog
-version:        0.1.0.0
+version:        0.2.0.0
```

Comment:
This doesn't appear to be a breaking change - are you willing to do a 0.1.0.1 version here? It seems like just a bugfix to provide proper output to me

Reply:
```suggestion
version:        0.1.1.0
```